React/my-app/src/redux/book/actions.js

```diff
@@ -68,25 +88,52 @@ export const resetGalleryView = () => {
   };
 };
 
-export const getBook = id => {
+export const createNewComment = (bookId, body) => {
   return async dispatch => {
     try {
-      const response = await getBookInfo(id);
+      debugger;
+      dispatch(uploadingComment(true));
+      const response = await postComment(bookId, body);
       if (responseOK(response)) {
```

Comment:
This is a comment from the previous pr, why do you need to check the response status?
isn't then always an ok response?

Reply:
Nacho told me to evaluate the case in which the server crash, so if it happens the app would be prepare to save the error.